src/strategies/dss-vest-balance-and-unpaid/index.ts

```diff
@@ -0,0 +1,37 @@
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+import { strategy as dssVestUnpaidStrategy } from '../dss-vest-unpaid';
+
+export const author = 'espendk';
+export const version = '1.0.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const score = await dssVestUnpaidStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    { address: options.dssVestAddress, ...options },
+    snapshot
+  );
+  const mgvScore = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    { address: options.tokenAddress, ...options },
+    snapshot
+  );
+
+  for (const address of Object.keys(score)) {
+    score[address] += mgvScore[address];
+  }
```

Comment:
Hey @espendk You can use dss-vest-unpai and erc20-balance-of in your space, result will be sum of those two strategies, let me know if it doesn't work for you

Reply:
Thanks for the suggestion :-) The thing is, we wish the UI to show the sum of these tokens as one, not two tokens.
If we use `dss-vest-unpaid` and `erc20-balance-of` as separate strategies in our space, the UI shows them separately:

<img width="472" alt="billede" src="https://github.com/snapshot-labs/snapshot-strategies/assets/3458856/e8daf4de-7fba-4a6b-b9f8-bb8964ced911">

The reason this is important is that we'll be adding two other strategies with separate symbols (TBD, but maybe `Builders` and `Pods`) and we'd like the UI to show eg `2 MGV + 3 Builders + 5 Pods`.

Reply:
you will be using both `dss-vest-unpaid` and `dss-vest-balance-and-unpaid` in your space?

Reply:
I suspect that we'll only be using `dss-vest-balance-and-unpaid`, but it's not clear at this time.

I mainly factored `dss-vest-unpaid` out into a separate strategy as it seemed generally useful to other projects that use DssVest for vesting. And those projects may not wish to mix it with `erc20-balance-of` like we do.